oap-server/server-bootstrap/src/main/resources/lal/envoy-als.yaml

```diff
@@ -17,24 +17,22 @@ rules:
   - name: envoy-als
     dsl: |
       filter {
-        json {
-        }
         // only collect abnormal logs (http status code >= 300, or commonProperties?.responseFlags is not empty)
-        if (parsed?.response?.responseCode as Integer < 400 && !parsed?.commonProperties?.responseFlags) {
```

Comment:
Why `!parsed?.commonProperties?.responseFlags` removed?

Reply:
Do you mean `< 400` means no internal error?

Reply:
Added back